.travis.yml

```diff
@@ -4,17 +4,10 @@ language: python
 python: 3.7
 services: docker
 env:
-- OS_VERSION=bionic GIT_URL=https://github.com/ceph/ceph            GIT_REF=octopus          IMG="popperized/ceph-builder:octopus-bionic"
-- OS_VERSION=bionic GIT_URL=https://github.com/ceph/ceph            GIT_REF=nautilus         IMG="popperized/ceph-builder:nautilus-bionic"
-- OS_VERSION=bionic GIT_URL=https://github.com/ceph/ceph            GIT_REF=mimic            IMG="popperized/ceph-builder:mimic-bionic"
-- OS_VERSION=bionic GIT_URL=https://github.com/uccross/skyhook-ceph GIT_REF=skyhook-luminous IMG="popperized/ceph-builder:skyhook-bionic"
+- OS_VERSION=7.8.2003 GIT_URL=https://github.com/uccross/skyhookdm-ceph GIT_REF=skyhook-luminous IMG="jcnitdgp25/ceph-builder:skyhook-centos"
```

Comment:
is intended to build the nautilus branch? or the luminous one?

Reply:
This is for luminous. Working on debugging nautilus ceph-builder image currently. Had some errors.